src/main.ts

```diff
@@ -434,21 +434,21 @@ getRadioButtonSetElement.forEach((radioButton: HTMLInputElement) => {
 dropDownElements.forEach((dropDown) => {
   // add click event listener to the dropdown parent element
   dropDown.addEventListener('click', (e) => {
-    // keep dropdown open when a subitem is clicked on
     e.stopPropagation();
     const target = e.target as HTMLElement;
-    if (!dropDown.children[0].innerHTML.includes(target.outerHTML)) return;
 
     const listElement = dropDown.lastElementChild as HTMLElement;
     if (listElement.id === 'showList') {
       listElement.id = '';
       return;
     }
 
-    // clear other open dropdown menus
+    // Clear other open dropdown menus
     dropDownElements.forEach((dropdown) => {
-      const listElement = dropdown.lastElementChild as HTMLElement;
-      listElement.id = '';
+      const otherListElement = dropdown.lastElementChild as HTMLElement;
+      if (otherListElement.id === 'showList') {
+        otherListElement.id = '';
+      }
```

Comment:
what does this do?

Reply:
This line sets the id attribute of the otherListElement to an empty string which will effectively close or open the drop-down.

Reply:
and how does it do that?

Reply:
I am a another Pull Request. I guess I messed in the git cz part when making this PR.

Reply:
never mind, I will fix them both myself

Reply:
```suggestion
       const listElement = dropdown.lastElementChild as HTMLElement;
       listElement.id = '';
```

Reply:
Thank you so much, will surely take care from next time. Always open for suggestions and improvements.